pkg/api/types.go

```diff
@@ -1414,8 +1412,13 @@ type Toleration struct {
 	// Effect indicates the taint effect to match. Empty means match all taint effects.
 	// When specified, allowed values are NoSchedule and PreferNoSchedule.
 	Effect TaintEffect `json:"effect,omitempty"`
-	// TODO: For forgiveness (#1574), we'd eventually add at least a grace period
-	// here, and possibly an occurrence threshold and period.
+	// ForgivenessSeconds Represents the period of time the toleration tolerates the taint.
```

Comment:
For the record, I don't want to unnecessarily introduce a new distinct concept. It should just be a property of the toleration.

Also, AFAIK, we currently don't have any fields containing the substring "duration". Let's also not introduce another time-related noun if at all possible.

FWIW, though it's not very relevant, we are planning to represent all disruptions as taints, and may introduce a frequency bound in the future. An example where frequency would be useful: it may be ok for my pod to tolerate a 30-minute disruption once a day, but not every 45 minutes.